kie-wb-smoke-tests/pom.xml

```diff
@@ -349,6 +354,35 @@
     </profile>
 
     <profile>
+      <!-- Enable selenium-based smoke tests to run. E.g.
+           mvn clean install -Pwildfly8x,selenium  -->
+      <id>selenium</id>
+      <activation>
+        <property>
+          <name>selenium</name>
+        </property>
+      </activation>
+      <properties>
+        <deployable.groupId>org.kie</deployable.groupId>
+        <deployable.artifactId>kie-wb-distribution-wars</deployable.artifactId>
+        <deployable.context>kie-wb</deployable.context>
```

Comment:
These properties should not be needed here. It should be enough to activate the specific war profile (kie-wb, kie-drools-wb).